BrightID/src/components/EditProfile/socialMediaThunks.ts

```diff
@@ -32,7 +31,8 @@ export async function linkSocialMediaApp(appId: string, appUserId: string) {
       Date.now() - signedTimestamp > SOCIAL_MEDIA_SIG_WAIT_TIME
     ) {
       try {
-        linked = await linkAppId(appId, appUserId, true);
+        // TODO
```

Comment:
This also seems to be missed to implement.
Linking social media apps do not need sponsoring or user request and do not follow general linking routine.
One of the things seems should not be applied to this linking is this check in this beginning of the `linkAppId`:
`if (applinkingStep !== app_linking_steps.SPONSOR_SUCCESS) {

Reply:
Social media registry app linking is now refactored.